e2e/testdata/fn-render/validate-generated-resource/db/Kptfile

```diff
@@ -0,0 +1,17 @@
+apiVersion: kpt.dev/v1alpha2
```

Comment:
We can avoid the sub package (`db`) and move the test case logic to the top level pkg. WDYT ?

Reply:
done